### PR TITLE
more consoles make appropriately colored light now

### PR DESCRIPTION
**Pull request by PKPenguin321**
>start up test server
>use break all lights verb
>remaining lights are pretty and colorful
>except for the requests console which is just normal white light

oh and also cargo consoles were also all white so i fixed those too

image
image2
imageCargo

### CODE REVIEW

**Comment by ChangelingRain** [comment]
use hex codes or the light color defines!

**Comment by KorPhaeron** [comment]
I don't think we should make the automatic revert button not work for /vg/ lighting until see if the memory leak is actually gone

File: code/modules/cargo/console.dm [hunk 1]
```diff
@@ -8,6 +8,8 @@
 	var/safety_warning = "For safety reasons the automated supply shuttle \
 		cannot transport live organisms, classified nuclear weaponry or \
 		homing beacons."
+	
+	light_color = "#E2853D"//orange
```
Comment (ChangelingRain) [inline code/modules/cargo/console.dm]: LIGHT_COLOR_BROWN

File: code/game/machinery/requests_console.dm [hunk 2]
```diff
@@ -68,7 +68,7 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 	if(stat & NOPOWER)
 		set_light(0)
 	else
-		set_light(2)
+		set_light(2,0.7,"#34D352")//green light
```
Comment (ChangelingRain) [inline code/game/machinery/requests_console.dm]: LIGHT_COLOR_GREEN

Comment (PKPenguin321) [inline code/game/machinery/requests_console.dm]: Intentionally shaded slightly different in this case

**Comment by KorPhaeron** [comment]
Lighting is busted

**Comment by PKPenguin321** [comment]
Yo they fixed it reopen

File: code/modules/cargo/console.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/cargo/console.dm]: >brown

That ain't orange

**Comment by KorPhaeron** [comment]
dont tell me what to do

**Pull request merged**